SpotifyAPI.Web/Models/SimpleTrack.cs

```diff
@@ -5,6 +5,9 @@ namespace SpotifyAPI.Web.Models
 {
   public class SimpleTrack : BasicModel
   {
+    [JsonProperty("album")]
+    public SimpleAlbum Album { get; set; }
+
```

Comment:
Small rant: Their docs are so inconsistent, it's really annoying at this point.

On this page: https://developer.spotify.com/documentation/web-api/reference/playlists/get-playlists-tracks/

It says 3 different things:

> On success, the response body contains an array of [track objects](https://developer.spotify.com/documentation/web-api/reference/object-model/#track-object-simplified) and episode objects

Where the link goes to track-object-simplified, containing NO album property

> track object (full)

This is documented on the page although it's theoretically never used in the call?

> playlist track object

Contains a link to track-object-full.